app-crypt/trousers/files/tmpfiles.d/trousers.conf

```diff
@@ -0,0 +1 @@
+C /var/lib/tpm/system.data 0644 tss tss - /usr/share/trousers/system.data
```

Comment:
Does this need to be recopied during updates or should it never be touched after it is first placed there?